services/appnavstate/api/src/main/kotlin/io/element/android/services/appnavstate/api/AppNavigationStateService.kt

```diff
@@ -24,6 +25,9 @@ import kotlinx.coroutines.flow.StateFlow
 
 interface AppNavigationStateService {
     val appNavigationStateFlow: StateFlow<AppNavigationState>
+    val appIsInForeground: StateFlow<Boolean>
```

Comment:
Maybe encapsulate so we only have one StateFlow?
Something like that : (not sure about the naming)
```
data class AppNavigationState(
    val isInForeground: Boolean,
    val currentNavigationStep: NavigationStep 
)
```